package.nls.json

```diff
@@ -1,4 +1,5 @@
 {
+  "command.activateIcons.title": "Activate VSCode Icon Theme",
```

Comment:
Honestly, I was troubled too. Took me some time to decide but I believe I chose the best solution in our case.